service-map/src/main/java/fi/nls/oskari/map/layer/OskariLayerServiceIbatisImpl.java

```diff
@@ -301,7 +303,71 @@ public OskariLayer find(int id) {
         LOG.debug("Parse layers with positive update rate sec took:", t2-t1, "ms");
         return layers;
     }
+    /**
+     * Returns the map layers which belong to the given parent.
+     * FIXME: Quick and dirty
+     * @param groupId
+     * @return layers of the given group
+     */
+    public List<OskariLayer> findAllByGroupId(final int groupId) {
```

Comment:
This is correctly marked as quick and dirty. :) If it's causing performance issues we should replace it.